$MirOS: ports/print/cups/patches/patch-cups_http-support_c,v 1.2 2006/11/08 18:25:44 bsiegert Exp $
--- cups/http-support.c.orig	Sat Jul 12 00:48:26 2008
+++ cups/http-support.c	Tue Oct 21 22:56:23 2008
@@ -672,7 +672,7 @@ httpGetDateString2(time_t t,		/* I - UNI
   tdate = gmtime(&t);
   snprintf(s, slen, "%s, %02d %s %d %02d:%02d:%02d GMT",
            http_days[tdate->tm_wday], tdate->tm_mday,
-	   http_months[tdate->tm_mon], tdate->tm_year + 1900,
+	   http_months[tdate->tm_mon], (int)tdate->tm_year + 1900,
 	   tdate->tm_hour, tdate->tm_min, tdate->tm_sec);
 
   return (s);
